renderdoc/driver/gl/gl_renderstate.cpp

```diff
@@ -879,16 +932,18 @@ void GLRenderState::FetchState(void *ctx, WrappedOpenGL *gl)
     m_Real->glGetFloatv(eGL_VIEWPORT, &Viewports[0].x);
     m_Real->glGetIntegerv(eGL_SCISSOR_BOX, &Scissors[0].x);
     Scissors[0].enabled = (m_Real->glIsEnabled(eGL_SCISSOR_TEST) == GL_TRUE);
-    m_Real->glGetDoublev(eGL_DEPTH_RANGE, &DepthRanges[0].nearZ);
+    if(!IsGLES)
+      m_Real->glGetDoublev(eGL_DEPTH_RANGE, &DepthRanges[0].nearZ);
 
     for(GLuint i = 1; i < (GLuint)ARRAY_COUNT(Viewports); i++)
       memcpy(&Viewports[i], &Viewports[0], sizeof(Viewports[i]));
 
     for(GLuint i = 1; i < (GLuint)ARRAY_COUNT(Scissors); i++)
       memcpy(&Scissors[i], &Scissors[0], sizeof(Scissors[i]));
 
-    for(GLuint i = 1; i < (GLuint)ARRAY_COUNT(DepthRanges); i++)
-      memcpy(&DepthRanges[i], &DepthRanges[0], sizeof(DepthRanges[i]));
+    if(!IsGLES)
```

Comment:
It's just because the line associated with the comment didn't change (only the lines below). It's harmless, once I approve the changes that's what matters, not the outdated diff comments disappearing.